pymc3/variational/inference.py

```diff
@@ -19,6 +19,7 @@
     'ADVI',
     'FullRankADVI',
     'SVGD',
+    'ASVGD',
```

Comment:
Amortized Stein Variational Gradient Descent 
Can be found [here](http://bayesiandeeplearning.org/papers/BDL_21.pdf). It is a novel black box algorithm for *wild inference* that doesn't require q(z)